.gitignore

```diff
@@ -31,6 +31,11 @@ mrdc.his
 # FPGA splice intermediate files
 *.jou
 
+# FPGA files for splicing a local bitstream
+hw/bitstream/local_chip_earlgrey_cw310.bit
```

Comment:
you won't need these if you use @a-will's approach below

Reply:
Yep, I've dropped the entire commit that changes these files